src/main/java/sirius/biz/scripting/CustomEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.scripting;
+
+import sirius.kernel.health.HandledException;
+
+import java.util.Optional;
+
+/**
+ * Provides a base class for all custom events handled by a {@link CustomEventDispatcher}.
+ */
+public abstract class CustomEvent {
```

Comment:
**Optional:** In my opinion the class name itself (and also `CustomEventDispatcher`) doesn't reflect that it is related to the scripting capabilities and could be anything. I would prefer something like `ScriptingEvent`.